pandunia-lekse.csv

```diff
@@ -1671,19 +1682,19 @@ lenxa||||practice (to exercise)||practicar (ensayar, ejercitar)|||||||||||||ekze
 lenxe|||zho:练习 (liànxí), jpn:練習 (renshū), kor:련습 (ryeonseup)|exercise||ensayo (ejercitio)|||||||||||||ekzerco|harjoitus|ćwiczenie
 lete|mosim||fra:l'été, rus:лето (leto)|summer|été|verano||лето|||||||||||somero|kesä (suvi)|lato
 letise|||eng:lettuce, jpn:レタス (retasu), hau:latas, hin:लेटिष (leṭiṣ), ben:লেটুস (leṭus), fra:laitue|lettuce|laitue|lechuga|alface|салат-латук||莴苣|レタス|||लेटिष|লেটুস||||laktuko|lehtisalaatti|sałata
-lewo|||tha:แล้ว /lɛːw/, zho:了 (le), wuu:了 (lə'), hak:了 (le), yue:了 (liu)|already (completed action)||ya (acción completado)|||||||||||||jam (esti farita)|jo (tehty teko)|już (skończona czynność)
+lewo|||tha:แล้ว (lɛːw), zho:了 (le), wuu:了 (lə'), hak:了 (le), yue:了 (liu)|already (completed action)||ya (acción completado)|||||||||||||jam (esti farita)|jo (tehty teko)|już (skończona czynność)
 leze|||zho:梨子 (lízi), yue:梨 (lei42), vie:lê,|pear||pera||||梨|||||||||piro|päärynä|gruszka
 liba||||erect||erigir (montar, armar)||||||||||||||nostaa pystyyn|postawić (stawiać)
 libania|dex|LB||Lebanon||Líbano||||||||||||||Libanon|Liban
-libele||||dragonfly|libellule|libélula|libélula|стрекоза||蜻蜓|蜻蛉|잠자리||व्याध पतंग|ফড়িং|capung||yusufçuk|libelo|sudenkorento|ważka
+libele|||spa:por:libélula, fra:libellule, deu:Libelle|dragonfly|libellule|libélula|libélula|стрекоза||蜻蜓|蜻蛉|잠자리||व्याध पतंग|ফড়িং|capung||yusufçuk|libelo|sudenkorento|ważka
 liberia|dex|LR||Liberia||Liberia||||||||||||||Liberia|Liberia
 libi||||erect (standing)||erguido||||||||||||||pysty (seisova)|postawiony (stojący)
 libia|dex|LY||Libya||Libia||||||||||||||Libya|Libia
 libobanze||||shelf (bookshelf)|étagère|estante (repisa)|prateleira (estante)|полка||架子 (搁板)|棚||||||||||
 libobaze||||stand (rack)||soporte (perchero, atril)||||架子|台||||||||||
 libu|||zho:立 (lì), yue:(lip), vie:lập, kor:립 (rib)|stand||estar de pie|||||立つ|||||||||seistä (seisoa)|stać
 lice|||zho:荔枝 (lìzhī), hin:लीची (līcī), eng:lychee, rus:личи (liči), may:leci, vie:lệ chi|lychee (litchi)||lichi||||||||||||||litši|liczi
-lida||||lead (direct)||dirigir (mandar, conducir, capitanear)||||||||||||||johtaa|prowadzić, dowodzić, kierować
+lida|||eng:lead, por:liderar|lead (direct)||dirigir (mandar, conducir, capitanear)||||||||||||||johtaa|prowadzić, dowodzić, kierować
```

Comment:
Is this etymology needed? The real etymology is the backderivation from "lider"! In addition, double etymologies harm the statistics.